mteb/tasks/Retrieval/multilingual/RuSciBenchRetrieval.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+from mteb.abstasks.MultilingualTask import MultilingualTask
+from mteb.abstasks.TaskMetadata import TaskMetadata
+from mteb.tasks.Retrieval.multilingual.NeuCLIR2023Retrieval import load_neuclir_data
+
+_LANGUAGES = {
+    "ru": ["rus-Cyrl"],
+    "en": ["eng-Latn"],
+}
+
+
+class RuSciBenchCiteRetrieval(AbsTaskRetrieval, MultilingualTask):
+    ignore_identical_ids = True
+
+    metadata = TaskMetadata(
+        name="RuSciBenchCiteRetrieval",
+        dataset={
+            "path": "mlsa-iai-msu-lab/ru_sci_bench_cite_retrieval",
+            "revision": "6cb447d02f41b8b775d5d9df7faf472f44d2f1db",
+        },
+        description="Retrieval of related scientific papers based on their title and abstract",
```

Comment:
```suggestion
        description="Retrieval of related scientific papers based on their title and abstract.",
```

Good. Could potentially add some information on the paper's origin.

I would especially like to have enough information to distinguish these from the below